sdk/python/foundation-models/system/distillation/conversation/distillation_conversational_task.ipynb

```diff
@@ -623,7 +623,129 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Cleanup Endpoint\n",
+    "## 5. (Optional) Disitlled Model Benchmarking\n",
+    "\n",
+    "1) Create connection to submit benchmarking job"
```

Comment:
ig, you can use H3 headers here and use 5.1 similar to what's followed above in the notebooks